### ISSUE TITLE
[Suggestion] camera filters/modes

### ISSUE THREAD

**Issue by AlienXtream** [tsunamayo/Starship-EVO#2172]
### Short Description of the issue:

camera modes like thermal scan, radiation scan (example)

thermal would create a... well... thermal camera like effect. heat producing blocks would show up brightest (based off their heat output), hull and other heat dissipators dimmer and such. depowered ships would still show up but be marginally brighter than the void of space. this would be a false image as actually creating a true thermal camera would be taxing on the render thread.

another idea i had in a similar vein is a radiation view, allowing the user to see both the radiation output of reactors and the trail of radiation left by ships. this one will probably be problematic as it would require the game creating those path's and remembering them but it could lead to some interesting search and rescue type quests.

**Comment by tsunamayo** [comment]
Yes I dig that, but it have to be linked to some gameplay element first. I wanted a "scanner" brick that allows you to lock a sub-system of the ship, so this could indeed be the job of that view.

**Comment by AlienXtream** [comment]
that indeed would be a better solution. i was just quickly sharing the idea before i forgot it alltogether
